content/applications/hr/recruitment/new_job.rst

```diff
@@ -0,0 +1,220 @@
+=============
+Job positions
+=============
+
+In the default *Recruitment* dashboard view, all job positions are shown, regardless of status.
```

Comment:
From our meeting we had on Guilabels, I don't think this would get one. The name dashboard isn't truly "official", I was calling it the dashboard since that just made sense- and you don't click on anything really, you are clicking on the Recruitment app tile from the main Odoo page, and you are taken here.

Reply:
Think in this case we're just saying the app name in a general sense, so italics is fine.

If were're suggesting (or hinting) to navigate here, we can also do it like this below. This is what I personally do because I think there's an implication to navigate here:
```suggestion
In the default :menuselection:`Recruitment` dashboard view, all job positions are shown, regardless of status.
```
If `Recruitment` is the name of the dashboard (the text heading is on the UI), a gui works too, however I don't think that's as useful, as say, calling out specific button link text during an active instruction.

All three seem to work, they just have different connotations. Up to you @larm-odoo 😉

Reply:
OK, all things being equal, I went with menuselection @StraubCreative, since you do click on it, in some fashion at some point, so I went with that. All good now!